test/integration/request-flow/request-response-error.test.js

```diff
@@ -0,0 +1,93 @@
+var expect = require('chai').expect,
```

Comment:
Test for history object in `requester-spec` test suite... asserting its structure and existence in case of error and success.